modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryDataRegion.java

```diff
@@ -169,14 +169,14 @@ public AtomicLong pageListCacheLimit() {
     /**
      * Calculates the size of segments in bytes.
      *
-     * @param dataRegionConfigView Data region configuration.
+     * @param size Data region size.
      * @param concurrencyLevel Number of concurrent segments in Ignite internal page mapping tables, must be greater than 0.
      */
     // TODO: IGNITE-16350 Add more and more detailed description
-    static long[] calculateSegmentSizes(PersistentPageMemoryDataRegionView dataRegionConfigView, int concurrencyLevel) {
+    static long[] calculateSegmentSizes(long size, int concurrencyLevel) {
```

Comment:
Why are these changes needed?

Reply:
This method actually needs just size, stick to the whole config looks redundant